roles/rac-db-setup/tasks/rac-db-install.yml

```diff
@@ -153,8 +153,8 @@
 - name: rac-db-install | opatch output
   debug:
     msg:
-      - "{{ apply_oneoff.cmd }}"
-      - "{{ apply_oneoff.stdout_lines }}"
+      - "{{ item.cmd }}"
```

Comment:
The changes to this task `rac-db-install | opatch output` may be backed out, as a [recent PR](https://github.com/google/bms-toolkit/pull/126/files#diff-2d3864b3dd75f1a76901aadadafa95259779a756c66469f79c2c278812f6a481) for EL8.5 fixed it.

Reply:
Yup, I will merge into current HEAD once approved.